odoo/models.py

```diff
@@ -5269,6 +5270,8 @@ def filtered(self, func):
         if isinstance(func, str):
             name = func
             func = lambda rec: any(rec.mapped(name))
+            # populate cache
+            self.mapped(name)
```

Comment:
Does this really makes a difference?

Reply:
Yes but maybe I should add a more descriptive comment